vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/apiserver/always_local_delegate_patch.go

```diff
@@ -0,0 +1,32 @@
+package apiserver
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// alwaysLocalDelegatePrefixes specify a list of API paths that we want to delegate to Kubernetes API server
+// instead of handling with OpenShift API server.
+var alwaysLocalDelegatePathPrefixes = sets.NewString()
```

Comment:
Wow, this got uglied up.  I've done it before too, so glass houses and all that.  Ok, until we're bit.